### ISSUE TITLE
Can I update the example app?

### ISSUE THREAD

**Issue by tyler-jewell** [iampawan/VelocityX#196]
First of all, this package is the best thing since chat GPT 3.5. It's as close to perfect as I have ever seen for a state management package. It makes writing Flutter apps so much more enjoyable. And I have tried Provider, Riverpod, BloC, GetX. I've tried it all. 

GREAT JOB.

That being said, I think the example app needs updated to truly showcase the power of the package. It seems like it may be a few versions behind and could use some updates. I have already developed a good app using a lot of VelocityX packages to train my AI agent and would not mind uploading to this repo in place of the current example. 

If you don't mind, can I push so code and get some comments on the updates?

**Comment by iampawan** [comment]
Thanks for your kind words. Please go ahead with a pull request :).

Note - I am working on a new library called "Rush" which will be even better than VelocityX with all it's power but only limited features which makes sense. Plus better docs, tests and more. Let me know if you would want to contribute. Thankyou

**Comment by tyler-jewell** [comment]
Would it be worth it to make updates to this repo? I have some suggestions as far as the repo setup. I use melos mono repo structure and can help you automate testing and such.

**Comment by iampawan** [comment]
Yes, it will be still maintained (at least by me ) unless Rush takes over and then also for some more time. It's a long way to go but once Rush is out, I will recommend people to use "Rush" over VelocityX.

**Comment by tyler-jewell** [comment]
Ok, let me add the example app and then play around with some other changes to get your tests back up and running so we can streamline some of the contributions.

**Comment by anderscheow** [comment]
@iampawan I'm willing to help you to work on `Rush` during my free time

**Comment by peter100u** [comment]
Rush where link url ? i need it

**Comment by peter100u** [comment]
> Rush where link url ? i need it

https://pub.dev/packages/rush should be this one.  I have started using it, but I hope to provide a GitHub link and more examples.  I am very excited about the arrival of this library.

**Comment by iampawan** [comment]
Hi @tyler-jewell @anderscheow @peter100u would love to get your help on Rush

**Comment by anderscheow** [comment]
@iampawan I'm always welcome to help during my free time.

**Comment by tylerpjewell-cat** [comment]
> Hi @tyler-jewell @anderscheow @peter100u would love to get your help on Rush

Send me link to package or repo.

**Comment by peter100u** [comment]
> > 你好@tyler-jewell @anderscheow @peter100u希望得到您对 Rush 的帮助
> 
> 请将包或仓库的链接发送给我。

https://github.com/iampawan/Rush
